beacon-chain/rpc/eth/config/handlers_test.go

```diff
@@ -353,7 +383,9 @@ func TestGetSpec(t *testing.T) {
 		case "TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH":
 			assert.Equal(t, "72", v)
 		case "TERMINAL_BLOCK_HASH":
-			assert.Equal(t, common.HexToHash("TerminalBlockHash"), common.HexToHash(v))
+			s, ok := v.(string)
```

Comment:
there's a bunch of new ones for deneb are they already added?

Reply:
I will update this endpoint in another PR, some things are missing